src/app/components/HomePageNonCritical.tsx

```diff
@@ -97,12 +98,18 @@ const HomePageNonCritical = () => {
       </section>
 
       <HeroContainer
-        className="uniprot-grid uniprot-grid--centered uniprot-grid--with-bleed supporting-data-section"
+        className={cn(
```

Comment:
why the extra logic?

Reply:
Initially to add the `styles[]` which in the end wasn't needed at that level, but also I think classnames makes it more readable - although happy to remove it